autoload/goimports.vim

```diff
@@ -10,6 +10,11 @@ if exists('g:goimports_simplify_cmd')
   let s:goimports_simplify_cmd = g:goimports_simplify_cmd
 endif
 
+let s:goimports_show_loclist = 1
+if exists('g:goimports_show_loclist')
+  let s:goimports_show_loclist = g:goimports_show_loclist
+endif
+
```

Comment:
Thank you. Please remove those lines.

Reply:
done, I juste thought it was your coding style to prepare vars (like import_cmd vars)